arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/AccountSnapshot.java

```diff
@@ -177,6 +177,17 @@ public AccountSnapshot incrementBalanceBy(Wei quantity) {
     return this;
   }
 
+  /**
+   * {@link AccountSnapshot#setBalanceToZero()} changes the balance of the AccountSnapshot to be
```

Comment:
CAn we have the same comment as others "Be
   * sure to work with a {@link AccountSnapshot#deepCopy} if necessary." bla bla ?

Reply:
Yes, will add

Reply:
Donc